code-bar-job-board-app/src/components/EditJob/index.js

```diff
@@ -63,7 +63,7 @@ const EditJob = ({updateJobPost, currentUser, isAdmin}) => {
             setFormJobPostLink(job.link_to_job);
 
             const jobClosingDate = document.getElementById('job-closing-date');
-            jobClosingDate.value = new Date(job.expiry_date.seconds * 1000).toLocaleDateString('en-CA');
+            jobClosingDate.value = new Date(job.expiry_date).toLocaleDateString('en-CA');
```

Comment:
Is there a reason for the Canada locale?

Reply:
Separate problem that the edit job form isn't reading the date either.  I'll fix that!

Reply:
So the form likes the date to be en-CA to set the value for the edit form.  Do you want me to look into it a bit further to find some better way of doing it?

Reply:
If you have time yes please :)